gradle/libs.versions.toml

```diff
@@ -26,6 +26,11 @@ spotless = "7.0.0.BETA3"
 androidx-test-core="1.5.0"
 androidx-test-junit="1.1.5"
 androidx-test-truth = "1.5.0"
+kotlinGradlePlugin = "1.9.24"
+coreKtx = "1.15.0"
+espressoCore = "3.6.1"
+appcompat = "1.7.0"
+material = "1.12.0"
```

Comment:
I'm not sure if we should change the version here, probably worth keeping the PR bundled only for the Crashlytics Version, we can always make those changes in another PR

Reply:
Will be definitely changing this and making it local to the test app